update.sh

```diff
@@ -1,6 +1,38 @@
 #!/bin/bash
 set -ue
 
+function usage() {
+  cat <<EOF
+
+  Update the node docker images.
+
+  Usage:
+    $0 [-s]
```

Comment:
Maybe explaining the version number format for `$0` would be useful

Reply:
What about
```
Usage: update.sh [MAJOR_VERSION] [OPTION]
Example: update.sh 8 -s
``

Reply:
Yeah, but it should also note that you can limit an update by `$0 [MAJOR_VERSION] [VARIANT] ` too:

```
  Usage:
    $0 [MAJOR_VERSION] [VARIANT] [-s]

  Examples:
    - update.sh         # Update all images
    - update.sh -s      # Update all images, skip updating Alpine and Yarn
    - update.sh 8.      # Update version 8 and variants (default, slim, alpine etc.)
    - update.sh 8 slim  # Update slim variant of version 8

  OPTIONS:
    -s Security update; skip updating the yarn and alpine versions.
    -h Show this message
```

Reply:
Excellent!!

Reply:
Nit pick but maybe `# Update all variant for version 8`